samples/cpp/benchmark_app/main.cpp

```diff
@@ -522,8 +589,12 @@ int main(int argc, char* argv[]) {
             }
         }
 
-        for (auto&& item : config) {
-            core.set_property(item.first, item.second);
+        // Property setting should be via the core::compile_model()
+        // rather than core::set_property() if target device is AUTO/MULTI.
```

Comment:
Yes. it was designed this way at beginning to avoid changing the hw device default configuration via `core::set_property("AUTO/MULTI", properties_contained_secondary_property)`. Because [`core::set_property()`](https://github.com/openvinotoolkit/openvino/blob/71c23489c6265e1f678a4dbc0e194e4ffc82c990/src/inference/src/ie_core.cpp#L993) will flatten the secondary property and change default configuration for corresponding hw device based on secondary properties passing into MULTI/AUTO. This [PR](https://github.com/openvinotoolkit/openvino/pull/13738) will fix this issue. So I revert this change here.